README.md

```diff
@@ -1,117 +1,166 @@
 
 ## Table of Contents
 
 1. [Description](#description)
 1. [Setup - The basics of getting started with puppet_bolt_server](#setup)
     * [What puppet_bolt_server affects](#what-puppet_bolt_server-affects)
-    * [Setup requirements](#setup-requirements)
-    * [Beginning with puppet_bolt_server](#beginning-with-puppet_bolt_server)
+1. [Installation - Step by step guide](#installation)
 1. [Usage - Configuration options and additional functionality](#usage)
 1. [Limitations - OS compatibility, etc.](#limitations)
-1. [Development - Guide for contributing to the module](#development)
 
 ## Description
 
-Briefly tell users why they might want to use your module. Explain what your
-module does and what kind of problems users can solve with it.
-
-This should be a fairly short description helps the user decide if your module
-is what they want.
+This module aims to configure a dedicated Puppet Enterprise compiler to become a Bolt Server, with the intention of offloading plan execution from Orchestrator on the Primary server, to Bolt on the Bolt server. A compiler is ideal because it already has access to Puppet Enterprise, code manager, and its local PuppetDB.
 
 ## Setup
 
-### What puppet_bolt_server affects **OPTIONAL**
+### What puppet_bolt_server affects
+
+The `puppet_bolt_server` module will perform the following activities:
+
+* Install Bolt on the node.
+* Create the `/root/.puppetlabs/etc/bolt/bolt-defaults.yaml` file with custom configuration to:
+    * Use the PCP transport
+    * Use the local PuppetDB for queries
+    * Consume a Puppet token
 
-If it's obvious what your module touches, you can skip this section. For
-example, folks can probably figure out that your mysql_instance module affects
-their MySQL instances.
+## Installation
 
-If there's more that they should know about, though, this is the place to
-mention:
+**Quickstart:** Configure the `puppet_bolt_server` using the PE Console
 
-* Files, packages, services, or operations that the module will alter, impact,
-  or execute.
-* Dependencies that your module automatically installs.
-* Warnings or other important notices.
+This setup will help you to quickly configure the `puppet_bolt_server` in your existing PE server.
 
-### Setup Requirements **OPTIONAL**
+1. Add the [puppetlabs-puppet_bolt_server](https://github.com/puppetlabs/puppetlabs-puppet_bolt_server) to your control repo.
+1. Add a new Node Group from the PE Console
 
-If your module requires anything extra before setting up (pluginsync enabled,
-another module, etc.), mention it here.
+```
+  Parent name: PE Infrastructure
+  Group name: Bolt Server
+  Environment: production
+```
 
-If your most recent release breaks compatibility or requires particular steps
-for upgrading, you might want to include an additional "Upgrading" section here.
+1. Add the class `puppet_bolt_server` to the Bolt Sever group created in the step above.
```

Comment:
This numbering needs to be fixed too, as the one below.

Reply:
Fixed!